core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+Abstractly, a BPI consists of
+* private messaging between agreement counterparties about the state or the requested or finalized state changes of the agreement between them,
+* the representation of a (commercial) agreement and its business rules and data as distinct workflows and worksteps between its counterparties organized into workgroups based on the stipulations of the (commercial) agreement
+* the deterministic processing and finalization of state change requests based on commercial documents between the agreement counterparties as stipulated by the (commercial) agreement.
+* the preservation of privacy of all agreement counterparties and their (commercial) data to other 3rd parties.
+
+BPIs are strongly dependent on the security and privacy capabilities of the DLT used to implement a BPI.
+
+**[R9]**	A BPI MUST be implemented on a DLT.
+
+Security and Privacy requirements of a BPI are key and are strongly dependent on the security and privacy assurances that the DLT on which the BPI is implemented can provide. BPIs need to take great care to avoid the following two situations:
+
+1. Weaken the security assurances of the underlying DLT by increasing the DLT attack surface. Such an expansion of the attack surface can occur through for example concentration of value-at-risk in one or more BPIs above the value used to economically secure the underlying DLT. This situation would make it economically attractive to attack, and subvert, the underlying DLT to extract the value in one or more BPIs.
+2.	Appreciably increase the existing attack surface of a DLT such that the security assurances of the BPI become significantly weaker than the underlying DLT. An example of such a situation can occur when a commercial state object such as a Financing contract or an Order in BPI A is dependent on a commercial state object such as an invoice as collateral in BPI B, and when BPI B has weaker transaction finality assurances than either BPI A or the underlying DLT. In that scenario, the commercial state object in BPI A cannot provably rely on the invoice as collateral in BPI B since the invoice might be reverted, and it would then no longer be a suitable collateral.
+	
+Hence, we enumerate the following requirements below:
+
+**[R10]**	A BPI MUST have the same security assurances as the DLT used to implement it.
+
+**[R11]**	(Commercial) State changes of a BPI MUST be verifiable on the DLT used to implement it.
+
+Verifiable in this context means that a 3rd party can verify through a cryptographic proof on the DLT that a transaction changed the state of a (commercial) state object in the BPI correctly based on agreed upon business rules - for example changing the Order status from open to completed.
```

Comment:
can verify, via a cryptographic proof on the DLT, that a ...

Reply:
correctly, based on agreed

Reply:
done